plasma_framework/contracts/src/exits/payment/PaymentExitGame.sol

```diff
@@ -14,28 +14,37 @@ import "../../utils/OnlyFromAddress.sol";
  * @notice The exit game contract implementation for Payment Transaction
  */
 contract PaymentExitGame is IExitProcessor, OnlyFromAddress, PaymentStandardExitRouter, PaymentInFlightExitRouter {
-    PlasmaFramework private plasmaFramework;
+    PaymentExitGameArgs.Args private paymentExitGameArgs;
+    bool private initDone = false;
 
     /**
      * @dev use struct PaymentExitGameArgs to avoid stack too deep compilation error.
      */
     constructor(PaymentExitGameArgs.Args memory args)
         public
-        PaymentStandardExitRouter(args)
-        PaymentInFlightExitRouter(args)
+        PaymentStandardExitRouter()
+        PaymentInFlightExitRouter()
     {
-        plasmaFramework = args.framework;
-
+        paymentExitGameArgs = args;
         // makes sure that the spending condition has already renounced ownership
         require(args.spendingConditionRegistry.owner() == address(0), "Spending condition registry ownership needs to be renounced");
     }
 
+    function init() public
+    {
```

Comment:
minor: the convention we are following is to put `{` on the line above.

https://solidity.readthedocs.io/en/v0.5.3/style-guide.html#function-declaration